src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -47,13 +48,16 @@
     private static final String MULTIPLOID_DATA_HG37 = largeFileTestDir + "gvcfs/HapMap5plex.ploidy10.b37.g.vcf";
     private static final String NA12878_HG37 = " src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testGVCFMode.gatk4.g.vcf";
     private static final String ARTIFICIAL_PHASED = getTestDataDir() + "/ArtificalPhasedData.1.g.vcf";
+    private static final String HG_00268_withSpaces = largeFileTestDir + "gvcfs/HG00268.spaceInSampleName.g.vcf";
```

Comment:
Use THIS_STYLE for names of constants. 

Also, why does this need to be a large file? Can't we test this functionality with a smaller file?

Reply:
It doesn't need to be large, indeed it isn't a particularly large file, I just figured it made sense to keep it with the other files (considering that this is just a subsection of one of the other files)